.github/workflows/build_test_deploy.yml

```diff
@@ -74,6 +74,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       NODE_OPTIONS: '--unhandled-rejections=strict'
+      npm_package_engines_node: 'v12.18.4' # HACK: temp fix for sharp@0.26.1
```

Comment:
We can remove this workaround when the next version of sharp is released.

See https://github.com/lovell/sharp/issues/2382 and https://github.com/lovell/sharp/pull/2379